src/hooks/useDefaultHandleApiError.ts

```diff
@@ -13,11 +22,26 @@ export function useDefaultHandleApiError() {
   return useCallback(
     (e: unknown) => {
       console.error(e);
-      chat.addMessage({
-        text: "Sorry, I'm unable to respond at the moment. Please try again later!",
-        source: MessageSource.BOT,
-        timestamp: new Date().toISOString(),
-      });
+      chat.addMessage(defaultErrorMessage);
+    },
+    [chat]
+  );
+}
+
+
+/**
+ * Returns a default handler function for errors with the initial message. It will log the error and
+ * set the headless messages state to a default error message.
+ *
+ * @internal
+ */
+export function useDefaultInitialMessageError() {
+  const chat = useChatActions();
+
+  return useCallback(
+    (e: unknown) => {
+      console.error(e);
+      chat.setMessages([defaultErrorMessage]);
```

Comment:
this would only be used for the initial message error, so that would be ok. Although I think i will split up the CRs and focus on the rendering as Yen was looking into the actual cause of this issue.

Reply:
PR: https://github.com/yext/chat-ui-react/pull/44